unstructured/partition/docx.py

```diff
@@ -305,7 +305,7 @@ def _increment_page_number(self) -> Iterator[PageBreak]:
         """Increment page-number by 1 and generate a PageBreak element if enabled."""
         self._page_counter += 1
         if self._include_page_breaks:
-            yield PageBreak("")
+            yield PageBreak("", detection_origin="docx")
```

Comment:
lets put the string `"docx"` as a constant for this sub module (e.g., after import define a `DETECTION_ORIGIN="docx"`) and reuse below

Reply:
Also refactored other origins 😄